<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WImportMatrixTable</class>
 <widget class="QWidget" name="WImportMatrixTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>511</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="in_matrix">
     <property name="text">
      <string>Matrix size (100,2)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="b_convert">
     <property name="text">
      <string>Convert to Excel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="b_tab">
     <property name="text">
      <string>Edit Table</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="b_plot">
     <property name="text">
      <string>Preview Plot</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Resources/pyleecan.qrc"/>
  <include location="../../../Resources/pyleecan.qrc"/>
  <include location="../../../Resources/pyleecan.qrc"/>
 </resources>
 <connections/>
</ui>
